docs/contributing-devguide.md

```diff
@@ -77,6 +77,13 @@ Run the following command to run a complete build. The rendered assets are avail
 ```shell
 make docs
 
+# Python 3
```

Comment:
I think we can `make serve-docs` here, right?

Reply:
Yes, after testing the make commands. We need to `make docs` first, then we can `make serve-docs`.
I'll fix the commands in the guide, thanks for pointing it out. 😁

Reply:
Since we can start our website with `make serve-docs`, should I just remove the python simple server?

Reply:
Makes sense to remove them. Thanks @leewei05 !

Reply:
Just updated the fix, you're welcome. 😁